### PR TITLE
chore(broker): add default broker config

### LINKED ISSUE #4182: Default config location
**Description**

During our engine performance tests, we wanted to change some defaults in the broker.
We only get it working with setting the env. variable to the cfg location. This was for us a bit unexpected. The env. is a nice to have, but we would expected that the cfg file is normally always in the same place, in the config folder next to the bin folder. I would suggest that we still support a default cfg location, as we did before.

Either by setting the env variable in the broker script, if not specified OR some logic in the broker itself.

### PR DESCRIPTION
**Pull request by pihme**
## Description

Add default broker config. Rename other configuration files to *.template. Adjust documentation

## Related issues

closes #4182

## Pull Request Checklist

- [ X ] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [ X ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [ X ] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

**Comment by pihme** [comment]
@saig0 This PR is supposed to be part of the release, please confer with @npepinpe

**Comment by pihme** [comment]
@Zelldon Invited you to the review because you created the issue. It's not required that you do a formal review, but a sanity check would be nice.

File: dist/src/main/config/broker.yaml.template [hunk 1]
```diff
@@ -0,0 +1,287 @@
+# Zeebe Broker configuration file (without embedded gateway)
```
Comment (Zelldon) [inline dist/src/main/config/broker.yaml.template:1]: what is the difference to broker.standalone?

Comment (Zelldon) [reply dist/src/main/config/broker.yaml.template:1]: Ok the only difference is embedded gateway? I'm asking myself whether this is really necessary to have two separate files for that.

**Review by ChrisKujawa — COMMENTED** [review]
Cool thanks :+1: I think it looks good just a comment on the broker and broker.standalone file.

File: dist/src/main/config/broker.yaml.template (hunk 1 above)
Comment (pihme) [reply dist/src/main/config/broker.yaml.template:1]: That's what we decided to do. The one file is for people who just want to test Zeebe standalone, the other is for people who want to configure nodes in a cluster. The documentation also hints into that direction

**Review by miguelpires — APPROVED** [review]

**Pull request merged**